### PR TITLE
[#1057] Type conversation

### PR DESCRIPTION
**Pull request by jhflorey**
This PR addresses issue https://github.com/episphere/connect/issues/1057

### CODE REVIEW

File: utils/shared.js [hunk 1]
```diff
@@ -1546,7 +1546,7 @@ const getTemplateForEmailLink = (
     continueUrl,
     preferredLanguage = fieldMapping.english.toString()
 ) => {
-    return preferredLanguage === fieldMapping.spanish.toString()
+    return preferredLanguage.toString() === fieldMapping.spanish.toString()
```
Comment (we-ai) [inline utils/shared.js:1549]: No need to convert to strings if they both are numbers.

Comment (jhflorey) [reply utils/shared.js]: Solved

**Review by Gbarra9 — APPROVED** [review]

File: utils/notifications.js [hunk 2]
```diff
@@ -705,7 +705,7 @@ const sendEmailLink = async (req, res) => {
         const body = {
             message: {
                 subject:
-                    preferredLanguage ===
+                    preferredLanguage.toString() ===
```
Comment (we-ai) [inline utils/notifications.js:708]: No need to convert to strings if they both are numbers.

Comment (jhflorey) [reply utils/notifications.js]: Solved

**Review by we-ai — APPROVED** [review]

**Pull request merged**